Python/Product/Profiling/Profiling/StandaloneTargetView.cs

```diff
@@ -56,7 +56,7 @@ sealed class StandaloneTargetView : INotifyPropertyChanged {
                 )
             ).ToList();
 
-            _customInterpreter = new PythonInterpreterView(Strings.LaunchProfiling_OtherInterpreter, "", null);
+            _customInterpreter = new CustomPythonInterpreterView();
```

Comment:
Can you check if we're still using `_customInterpreter` anywhere other than this method? If not, it should become a local.

Reply:
It's only used inside this class and have been changed to a local variable.